smokes/e2e/reason_force.scenarios.coffee

```diff
@@ -0,0 +1,44 @@
+# coffee script 
+# test goal: checks the capability to define a reason and to cancel/start the build
+
+forceSchedulerPage = require('./reason_force.coffee')
+forcePage = null
+
+describe('', () ->
+    beforeEach(() ->
+        forcePage =  new forceSchedulerPage();
+        forcePage.goToDefaultURL();
+    )
+
+    describe 'forceandcancel', () ->
+        it 'should create a build with a dedicated reason and cancel it', () ->
+
+            forcePage.goToURL('#/builders/1/');
+            currentURL0 = browser.getCurrentUrl();
+            currentURL0.then (v) -> console.log v;
+            forcePage.goToURL('#/builders/1/force');
+            currentURL1 = browser.getCurrentUrl();
+            currentURL1.then (v) -> console.log(v);
+            forcePage.goToURL('#/builders/1/force/force');
+            forcePage.writeAndCheckReason("New Test Reason");
+            forcePage.writeAndCheckYourName("FaceLess User");
+            forcePage.writeAndCheckProjectName("BBOT9");
+            forcePage.writeAndCheckBranchName("Gerrit Branch");
+            forcePage.writeAndCheckRepo("http//name.com");
+            forcePage.writeAndCheckRevisionName("12345");
+            forcePage.clickOnCancelButton().click();
```

Comment:
maybe we should make sure this did not create a build?
make sure it goes back to the builder page?